index.html

```diff
@@ -6776,14 +6780,7 @@ <h1>JSON Schema for TD Instance Validation</h1>
                     "minimum": 0
                 },
                 "multipleOf": {
-                    "oneOf": [
-                        {
-                            "type": "integer"
-                        },
-                        {
-                            "type": "number"
-                        }
-                    ]
+                    "$ref": "#/definitions/multipleOfDefinition"
```

Comment:
Note that I didn't make this change, it was generated by the render script. Was this previously added in a source file without checking in the index.html output?

Reply:
Sorry, this change has been recently introduced by #1204. Maybe there could be a separate regeneration on the `main` branch to differentiate it from your changes.

Reply:
No problem. I can either rebase on top of main once this is done, or even just land the changes in this PR now I know they are desired.

Reply:
(I've found it very confusing which files are meant to be edited directly vs. generated so it's easily done!)